i18n/models.py

```diff
@@ -45,10 +45,17 @@ def internationalizable_fields(cls):
     def get_i18n_objects(cls):
         return cls.objects
 
+    @classmethod
+    def get_serializer(cls):
+        name = cls.__name__ + 'Serializer'
+        serializers = __import__('curricula.serializers', fromlist=[name])
```

Comment:
could you add a comment here linking to the SO answer that led you to this?

Reply:
Added in commit b154e2c9f029c8c223ed68968d6a5edf62107c38.